### ISSUE TITLE
404 - Provide a smooth experience for families that visit a nonworking link

### ISSUE THREAD

**Issue by mekline** [lookit/lookit-api#1117]
# TL;DR

Upcoming site merge will cause any existing links to various subdomains of childrenhelpingscience.com to fail, such as: 

https://childrenhelpingscience.com/parc <- a static page
https://childrenhelpingscience.com/all-studies/toy-room <- example of a single study's page

Families might also (somehow) currently click on a nonworking lookit.mit.edu subdomain link, and the resulting experience isn't particularly helpful. 

# Narrative

As a family, if I click on a link that doesn't work for whatever reason, I want to see a friendly helpful guide to places I might want to go, like signup or viewing studies, in place of the current "Not Found" screen. 

![Screen Shot 2023-02-17 at 12 09 32 PM](https://user-images.githubusercontent.com/6819829/219718493-48b58cbd-c170-416f-afec-d7bbe3c94b54.png)

# New Page design

Big text: Oops! We couldn't find that page!

"Big C" logo guy, for friendliness, on white background

Smaller text: Welcome to Children Helping Science! We recently relaunched our website, which means that some links may no longer be working. To find studies you can participate in from home, please click on one of the options below:

[4 age groups with big stock photos, identical to these images and links as they appear on the front page]

Rough mockup in the slide deck: https://docs.google.com/presentation/d/1s91BnmNT-vR6RqSeadoU2wB1c-wjkNBsJjS_bg_kLSw/edit#slide=id.g23b16ee5720_0_0

# Acceptance Criteria
A precise description of how/when this issue will be considered closed, e.g.:

- [ ] Decide format & content of new "404" type page!
- [ ] Implement new page
- [ ] Testing works for a nonworking lookit.mit.edu subdomain
- [ ] Potentially leave ticket open until post CHS merger to ensure that those pages in fact wind up failing in the way we want them to
